lp_ui/README.md

```diff
@@ -1,19 +1,56 @@
-## Prerequisites
+# Liquidity Provider UI
 
-- Docker
-- NodeJS v14+
+This UI can be used for creating and managing migration pools as defined by `solana/migration` and `ethereum/contracts/bridge/utils/Migrator.sol`
 
-Run the following from the root of this repo
+## Quick Start
 
 ```bash
-DOCKER_BUILDKIT=1 docker build --target node-export -f Dockerfile.proto -o type=local,dest=. .
-DOCKER_BUILDKIT=1 docker build -f solana/Dockerfile.wasm -o type=local,dest=. solana
-npm ci --prefix ethereum
-npm ci --prefix sdk/js
-npm run build --prefix sdk/js
+npm ci
+REACT_APP_CLUSTER=mainnet npm start
 ```
 
-The remaining steps can be run from this folder
+Navigate to http://localhost:3000/
+
+## Create a new Ethereum pool
+
+> Please ensure your wallet is connected to the desired chain! These instructions are suitable for any EVM chain.
+
+1. Click "Create a New Ethereum Pool".
+1. Click "Connect" to connect your Metamask.
```

Comment:
Are these really all "1."?  Or is it git rendering of the MD file?

Reply:
This is (one) markdown syntax for ordered lists, and renderers will handle the numbering. 🙂 https://www.markdownguide.org/basic-syntax/#ordered-lists